Python/pywarpx/picmi.py

```diff
@@ -901,6 +901,15 @@ class ElectromagneticSolver(picmistandard.PICMI_ElectromagneticSolver):
     warpx_psatd_do_time_averaging: bool, optional
         Whether to do the time averaging for the spectral solver
 
+    warpx_psatd_J_in_time: string, {'constant', 'linear'}, default='constant'
```

Comment:
In other places where there are multiple options like this, the `string` type is not specified. I don't have a strong opinion about this, just noting the inconsistency.

Reply:
Sounds good, I removed `string`, even though I personally think it's better to state the type explicitly.